src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverProjectExtension.kt

```diff
@@ -111,28 +131,27 @@ public interface KoverProjectExtension {
     }
 
     @Deprecated(
-        message = "XML report setting was moved to '$REGULAR_REPORT_EXTENSION_NAME { xml { ... } }'. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_6_TO_0_7}",
+        message = "XML report setting was moved to '$REPORT_EXTENSION_NAME { xml { ... } }'. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_6_TO_0_7}",
         level = DeprecationLevel.ERROR
     )
     public fun xmlReport(block: () -> Unit) {
     }
 
     @Deprecated(
-        message = "HTML report setting was moved to '$REGULAR_REPORT_EXTENSION_NAME { html { ... } }'. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_6_TO_0_7}",
+        message = "HTML report setting was moved to '$REPORT_EXTENSION_NAME { html { ... } }'. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_6_TO_0_7}",
         level = DeprecationLevel.ERROR
     )
     public fun htmlReport(block: () -> Unit) {
     }
 
     @Deprecated(
-        message = "Verification report setting was moved to '$REGULAR_REPORT_EXTENSION_NAME { verify { ... } }'. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_6_TO_0_7}",
+        message = "Verification report setting was moved to '$REPORT_EXTENSION_NAME { verify { ... } }'. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_6_TO_0_7}",
```

Comment:
Same here and in HTML: as far as I understood, everything should now be under `defaults` or `androidReports`?